addons/mail/static/src/core/common/message.xml

```diff
@@ -5,6 +5,7 @@
         <div t-if="message.isNotification" class="o-mail-NotificationMessage text-break mx-auto text-muted opacity-75 px-3 text-center smaller" t-on-click="onClickNotificationMessage"  t-att-class="props.className" t-ref="root">
             <i t-if="message.notificationIcon" t-attf-class="{{ message.notificationIcon }} me-1"/>
             <span class="o-mail-NotificationMessage-author d-inline" t-if="authorName and !message.body.includes(escape(authorName))" t-esc="authorName"/> <t t-out="message.body"/>
+            <span class="o-mail-NotificationMessage-date o-xsmaller ms-1 opacity-50" t-out="message.dateSimpleWithDay"/>
```

Comment:
Can use `o-mail-Message-date` rather than introduce `o-mail-Notification-date` classname, so that you reuse the `opacity: 50%` in SCSS.

You can also clean the `.dark.scss` as the rule is redundant with the non-dark scss (at some point the value were different but that's not the case anymore).